src/Components/Card/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { CardProps } from "./types";
+import { Container, CardStyle, Emoji, TodoContainer } from "./styles";
+
+
+const Card: React.FC<CardProps> = ({ todo }) => {
+
+    return (
+        <Container>
+            <CardStyle>
+            <TodoContainer>
+            {todo}
+            </TodoContainer>
+            <div>
+            <Emoji role="img" aria-label="trash-can">🗑</Emoji>
```

Comment:
Just be careful with using emojis, because they will be different in each browser and the may not work correctly for older browsers

Reply:
Yeah, I read about that in an article. I tried doing it the safest way possible but got stuck during the Typescript part. I might change the emojis to React Icons in the future. Thanks for the heads up Vic!